R-package/R/lgb.cv.R

```diff
@@ -71,8 +71,8 @@ CVBooster <- R6::R6Class(
 #' @export
 lgb.cv <- function(params = list()
                    , data
-                   , nrounds = 10L
-                   , nfold = 3L
+                   , nrounds = 100L
+                   , nfold = 5L
```

Comment:
```suggestion
                   , nfold = 3L
```

Please revert this change to `nfold`.

I'm open to updating the R package's defaults to match the defaults from LightGBM's core library (https://lightgbm.readthedocs.io/en/latest/Parameters.html), but the number of folds to use in k-fold cross-validation should be left to individual wrapper packages. It would be a nice side benefit to have closer alignment between the R and Python package but I don't support making breaking changes for only that purpose.

In other words, I support this PR not as "make the R package more closely match the Python package" but "make the R package more closely match the LightGBM core library".

I've changed the PR title accordingly.